crates/compilers/src/compilers/zksolc/settings.rs

```diff
@@ -76,6 +77,19 @@ impl ZkSolcSettings {
         Self { output_selection: output_selection.into(), ..Default::default() }
     }
 
+    /// Consumes the type and returns a [Settings::sanitize] version
+    pub fn sanitized(mut self, version: &Version) -> Self {
```

Comment:
```suggestion
    pub fn sanitized(mut self, solc_version: &Version) -> Self {
```
Would probably clarify here that version is the `solc_version` vs the zksolc one